src/frontend/komponenter/Felleskomponenter/Venstremeny/Link.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactChild } from 'react';
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+interface Props {
+    children: ReactChild;
+    id: string;
+    to?: string;
+    active?: boolean;
+    className?: string;
+}
+
+const Link = ({ active = true, id, to, children, className }: Props) => {
```

Comment:
why do we need the default value for active?

Reply:
Raw copy from helse. But I think it makes sense to have true as defaultvalue for links.